web3walletcore/src/commonMain/kotlin/com/sonsofcrypto/web3walletcore/services/actions/Action.kt

```diff
@@ -1,25 +1,28 @@
 package com.sonsofcrypto.web3walletcore.services.actions
 
-import com.sonsofcrypto.web3walletcore.services.actions.Action.Type.*
+import com.sonsofcrypto.web3walletcore.extensions.Localized
 
-data class Action(
-    val type: Type,
-    val imageName: String, // Security, Social, etc
+sealed class Action(
     val title: String,
     val body: String,
-    val deepLink: String,
```

Comment:
I feel here we should reuse the logic we have in deep links...would be less code in the long run and can be reused from multiple places in the app such as settings, etc rather than having to inject all the possible action wireframes to the dashboard...
(I miss understood the comment before)

Reply:
Navigation and Actions service are two separate responsibilities. Should not necessarily know about each other. Always keeping action handling in presenter and navigation execution in wireframe make is easier to navigate, understand and debug. Using deep like requires looking at deeplink code. Debugging deaplink string. If it is explicit in presenter and wireframe it is much easier to follow and debug